src/builder/QuarkBuilder.sol

```diff
@@ -982,6 +997,317 @@ contract QuarkBuilder {
         });
     }
 
+    struct MorphoBorrowIntent {
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
```

Comment:
The naming is inherited from the existing `CometBorrowIntent` 
And since I believe client is probably already using `CometBorrowIntent` to calling comet actions, probably it's best for us to keep the naming as consistent as the existing intent.   Or we might want to also migrate the existing comet intents too.

I guess when comet actions was created, `AssetSymbol` is treated as one single object, so there became `AssetSymbol` and `collateral_AssetSymbol`